### PR TITLE
Remove anti piracy messages

### PR DESCRIPTION
**Pull request by illusionyy**
Follow up to  #184

### CODE REVIEW

**Comment by Dacatzgoescodin5267** [comment]
So was it an troll who put the crap in to damage our ear drums?

**Comment by PublicVoidUpdate** [comment]
> So was it an troll who put the crap in to damage our ear drums?

Fucking stupid too, ISO are a freaking disk image file.

**Comment by TempleOfHorus** [comment]
Bought items on xbox live comes with xcp as original format. After download, I decrypt them and convert them to ISO. I don't see any reason why ISO format shouldn't be supported in legal way.

**Comment by PublicVoidUpdate** [comment]
If you play directly or indirectly from a disk its ISO too

**Review by ryzendew — APPROVED** [review]
thank you for restoring xenia and removing that garbage!

**Review by Troplo — APPROVED** [review]

File: src/xenia/emulator.cc
Comment (aaaaaa123456789) [inline src/xenia/emulator.cc]: There's two typos here: there should be a full stop after the end of the first sentence (before the newline) and it should be "guide", not "quide".

File: src/xenia/emulator.cc
Comment (aaaaaa123456789) [inline src/xenia/emulator.cc]: Might want to remove this entirely? It's still demeaning.

File: src/xenia/emulator.h
Comment (aaaaaa123456789) [inline src/xenia/emulator.h:227]: Are these flags still in use?

**Review by aaaaaa123456789 — COMMENTED** [review]
Just taking a look at the linked PR!

**Comment by EeveeEuphoria** [comment]
I think it would stand to be reasonable to remove other things this troll added to the emulator, like this insulting message. Especially due to the implications that Xenia is scanning the titles of other windows, which is not something I'd trust any program to be doing myself. (not my screenshot)
EDIT: I'm wrong, this opens up when you try to open a ZIP file, I'm dumb lol. Either way, message is still there, should be removed.
image

File: src/xenia/emulator.cc
Comment (Whovian9369) [inline src/xenia/emulator.cc]: It might be good to remove (or at least change) this message to remove this hostile message as well, since it much questions the user's intelligence.

**Comment by JPeisach** [comment]
Ok, let's stop offending each other and suggest solutions.

- Opening a browser and sending a troll error is definitely a no-no.
- AP messages technically are needed, but they shouldn't be nagging the user and calling them morons.
- consider the genuine user: what if I dumped my game and it's an iso? Considering how generally accepted the format is, there could be other ways to look for an image to be legit or not. Think of console keys, maybe enforce ISOs are encrypted and can only be decrypted with a console specific key?
- messages:
-- someone suggested this in the other PR, a startup message with a disclaimer about piracy and saying they won't be assisted in the discord
-- simpler messages: just say like, the file provided is invalid and give a link to help dumping actual games
- in the discord, try to make users know the rules more. Maybe have a channel ende users have to say they specifically accept and understand the emulator's purposes and that they will not be assisted for piracy before a moderator approves them into the server

- also consider: why go through this much effort? it's not like there's an emulator that has perfectly fixed the issue of pirates using it. Requiring console keys to import or some other way of having console specific information in order to use the emulator will probably be safe enough. And also, if Xenia wants to also assist in homebrew development, like how Dolphin makes it easy to load a .dol or .elf, then sometimes you just got to accept that pirates will always be around, as unfortunate as it is to say.

And, again, why was this considered? What was the problem in the first place? Was some sort of warning sent from Microsoft, or is there some legal grounds you guys are worried about? Or is the issue pirates in your discord server? Because I'm sure there are many people willing to help and pitch ideas, not flame others, bleed ears and call others morons.

**Comment by nzgamer41** [comment]
+1 for merging this. You can't control what people do on their computers, and the "measures" the troll used to "combat" piracy hurt the reputation of Xenia. Discord moderation issues should not interfere with a program used by people who aren't in said discord.

File: src/xenia/emulator.h
Comment (illusionyy) [reply src/xenia/emulator.h:227]: Not anymore. At least it's used in POSIX to return flag set

File: src/xenia/emulator.cc
Comment (illusionyy) [reply src/xenia/emulator.cc]: Maybe fixed

File: src/xenia/emulator.cc
Comment (illusionyy) [reply src/xenia/emulator.cc]: Maybe fixed

**Comment by Clybius** [comment]
Not gonna be using this software, and I'll make sure those who are curious about this project are aware of this scenario as well before heading into such a project. Malicious by design.

**Comment by xan1242** [comment]
One thing I find funny is how this is limited to Windows users: `#if XE_PLATFORM_WIN32 == 1`

~~It's fair tho, users of other platforms probably don't need this /s~~

**Comment by Spaceghost** [comment]
I signed up for the real Xenia teams patreon. I’m going to ask them to find a way to remove the committer and denounce Xenia-canary team. I actually just upped all my many emulator patreons and made sure you aren’t receiving a dime from me or my friends.

this will not be forgiven. I’ll happily put up to a thousand dollars per month for a year into any project that wants to replace this and the “moron” code. If ignored, I will be pursuing this on a direct and personal level instead of addressing the team over GitHub issues cordially

remove the core committer(s) who approved and authored this code and ban them from ever having commit bits again.

Edit: committer voluntarily left, ghosted his account. Core team remaining is cleaning code and hopefully arranging a public apology to the whole scene.

Xenia (not canary) team response: https://twitter.com/xeniaproject/status/1684568619265712129

**Comment by illusionyy** [comment]
> One thing I find funny is how this is limited to Windows users: `#if XE_PLATFORM_WIN32 == 1`
> 
> ~It's fair tho, users of other platforms probably don't need this /s~

Not sure if posix had implementation for message boxes, didn't check

**Comment by illusionyy** [comment]
Merging for now, sorry for inconvenience caused by #180.

**Comment by illusionyy** [comment]
Will follow up with another PR if valid feedback is given here.

**Comment by xan1242** [comment]
> > One thing I find funny is how this is limited to Windows users: `#if XE_PLATFORM_WIN32 == 1`
> > ~It's fair tho, users of other platforms probably don't need this /s~
> 
> Not sure if posix had implementation for message boxes, didn't check

Yeah but I'm thinking maybe implement this in imgui instead since the project is already using it.

In any case this is a non-issue.

**Comment by illusionyy** [comment]
Yea imgui is a good idea, I'll setup VS and add it in the future.

**Comment by Masamune3210** [comment]
> I signed up for the real Xenia teams patreon. I’m going to ask them to find a way to remove you and denounce you Xenia-canary team. I actually just upped all my many emulator patreons and made sure you aren’t receiving a dime from me or my friends.
> 
> this will not be forgiven. I’ll happily put up to a thousand dollars per month for a year into any project that wants to replace you and your “moron” code. If ignored, I will be pursuing this on a direct and personal level instead of addressing the team over GitHub issues cordially
> 
> remove the core committer(s) who approved and authored this code and ban them from ever having commit bits again.

Grr, I'm mad because of something that happened once, is fixed, didn't even last that long and already has steps in place to prevent in the future. No green paper for you grre

**Comment by xan1242** [comment]
https://github.com/xenia-canary/xenia-canary/pull/180#issuecomment-1653714061

I'd just do it this way as Silent suggested - a checkbox (like a license agreement).

Oh and persist this setting in the regular Xenia config too, don't use Windows Registry lol

**Pull request merged**
